extensions/ql-vscode/src/vscode-tests/no-workspace/contextual/queryResolver.test.ts

```diff
@@ -1,42 +1,63 @@
 import * as yaml from "js-yaml";
-import * as sinon from "sinon";
-import { expect } from "chai";
-import * as pq from "proxyquire";
+import * as fs from "fs-extra";
+
 import { KeyType } from "../../../contextual/keyType";
 import { getErrorMessage } from "../../../pure/helpers-pure";
 
-const proxyquire = pq.noPreserveCache().noCallThru();
+import * as helpers from "../../../helpers";
+import {
+  qlpackOfDatabase,
+  resolveQueries,
+} from "../../../contextual/queryResolver";
+import { CodeQLCliServer } from "../../../cli";
+import { DatabaseItem } from "../../../databases";
 
 describe("queryResolver", () => {
-  let module: Record<string, Function>;
-  let writeFileSpy: sinon.SinonSpy;
-  let getQlPackForDbschemeSpy: sinon.SinonStub;
-  let getPrimaryDbschemeSpy: sinon.SinonStub;
-  let mockCli: Record<
-    string,
-    sinon.SinonStub | Record<string, sinon.SinonStub>
-  >;
+  const writeFileSpy = jest.spyOn(fs, "writeFile");
+
+  const getQlPackForDbschemeSpy = jest.spyOn(helpers, "getQlPackForDbscheme");
+  const getPrimaryDbschemeSpy = jest.spyOn(helpers, "getPrimaryDbscheme");
+  jest.spyOn(helpers, "getOnDiskWorkspaceFolders").mockReturnValue([]);
+  jest.spyOn(helpers, "showAndLogErrorMessage").mockResolvedValue(undefined);
+
+  const mockCli = {
+    resolveQueriesInSuite: jest.fn(),
+    cliConstraints: {
+      supportsAllowLibraryPacksInResolveQueries: jest.fn(),
+    },
+  };
+
   beforeEach(() => {
-    mockCli = {
-      resolveQueriesInSuite: sinon.stub(),
-      cliConstraints: {
-        supportsAllowLibraryPacksInResolveQueries: sinon.stub().returns(true),
-      },
-    };
-    module = createModule();
+    writeFileSpy.mockReset().mockImplementation(() => Promise.resolve());
```

Comment:
How come we need to mockReset() every time? Is this not default behaviour?

Reply:
I've removed all of these calls and used `resetMocks: true` instead.